.github/workflows/phpunit-tests.yml

```
@@ -202,8 +202,8 @@ jobs:
       matrix:
         os: [ ubuntu-24.04 ]
         php: [ '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4' ]
-        db-type: [ 'mysql' ]
-        db-version: [ '9.4' ]
+        db-type: [ 'mysql', 'mariadb' ]
+        db-version: [ '9.4', '12.0.2' ]
```

```suggestion
        db-version: [ '9.4', '12.0' ]
```

I don't believe the `.2` is necessary. A `.3` rarely happens, but it can. Dropping the `.2` should result in any further bug fix releases being used automatically.